docs/src/components/home/Sponsors.tsx

```diff
@@ -4,16 +4,21 @@ import Container from '@mui/material/Container';
 import Typography from '@mui/material/Typography';
 import DiamondSponsors from 'docs/src/components/home/DiamondSponsors';
 import GoldSponsors from 'docs/src/components/home/GoldSponsors';
+import SectionHeadline from 'docs/src/components/typography/SectionHeadline';
+import GradientText from 'docs/src/components/typography/GradientText';
 
 const Sponsors = () => {
   return (
     <Container sx={{ py: { xs: 4, md: 8 } }}>
-      <Typography variant="h2" sx={{ my: 1 }}>
-        Our sponsors
-      </Typography>
-      <Typography color="text.secondary" sx={{ mb: { xs: 2, md: 4 }, maxWidth: 450 }}>
-        The continued development and maintenance of MUI is greatly helped by our generous sponsors.
-      </Typography>
+      <SectionHeadline
+        overline="Sponsors"
+        title={
+          <Typography variant="h2" sx={{ my: 1 }}>
+            <GradientText>You</GradientText> make this possible
+          </Typography>
+        }
+        description="The continued development of our open source tools is made possible by our generous sponsors."
```

Comment:
Not strictly true, but okay...

Reply:
We could say "made possible _in part_ by" but then it's a bit wordy and weaselly.

Reply:
?

```suggestion
        description="The development of these open-source tools is accelerated by our generous sponsors."
```